tests/area/test_area.py

```diff
@@ -46,88 +62,72 @@ def setup_method(self):
             "H1 Storage2": (23, 25),
         }
 
-        self.strategy = MagicMock(spec=StorageStrategy)
-        self.config = MagicMock(spec=SimulationConfig)
-        self.config.slot_length = duration(minutes=15)
-        self.config.tick_length = duration(seconds=15)
-        self.config.ticks_per_slot = int(self.config.slot_length.seconds /
-                                         self.config.tick_length.seconds)
-        self.config.start_date = today(tz=constants.TIME_ZONE)
-        GlobalConfig.sim_duration = duration(days=1)
-        self.config.sim_duration = duration(days=1)
-        self.config.grid_fee_type = 1
-        self.config.end_date = self.config.start_date + self.config.sim_duration
-        self.area = Area("test_area", None, None, self.strategy,
-                         self.config, None, grid_fee_percentage=1)
-        self.area_child = Area("test_area_c", None, None, self.strategy,
-                               self.config, None, grid_fee_percentage=1)
-        self.area_child.parent = self.area
-        self.area.children = [self.area_child]
-        self.area.grid_fee_percentage = 1
-        self.dispatcher = AreaDispatcher(self.area)
-        self.stats = AreaStats(self.area._markets, self.area)
-
-    def teardown_method(self):
+    @staticmethod
+    def teardown_method():
```

Comment:
This will work with pytest, however beware when stumbling upon existing unittest.TestCase tests. In the latter, setup and teardown methods have to be normal methods and not static methods